contents/blog/how-to-start-a-growth-team.md

```diff
@@ -0,0 +1,112 @@
+---
+date: 2023-07-26
+title: "How to start a growth team (as an engineer)"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ['lior-neu-ner']
+featuredImage: ../images/blog/experiment-hog.jpg
+featuredImageType: full
+category: Product growth
+tags: 
+  - Guides
+  - Product metrics
+---
+
+Fast-growing startups are like rocket ships, and as co-founder of Google, Eric Schmidt, once said: "If you're offered a seat on a rocket ship, don't ask what seat. Just get on.". In these rocket ships, it's the job of the growth team to provide the fuel for the journey by igniting user acquisition and retention. 
```

Comment:
Im 50/50 whether to include this paragraph or not. Thoughts?

Reply:
I think its fine to leave, but maybe get rid of the Eric Schmidt quote, I don't think it adds anything.